collector/src/test/java/com/google/cloud/trace/zipkin/ZipkinCollectorIntegrationTest.java

```diff
@@ -43,28 +43,13 @@
   @Test
   public void traceSinkGetsCalled() throws InterruptedException
   {
-    this.restTemplate.postForLocation("/api/v1/spans", createTestSpans(1l, 2l, 45l));
-    verify(traceSink, times(3)).receive(any(Trace.class));
-    verifyNoMoreInteractions(traceSink);
-  }
+    //LOTS_OF_SPANS is too much! I need at most 10!
+    final Span[] spans = Arrays.copyOfRange(LOTS_OF_SPANS, 0, min(10, LOTS_OF_SPANS.length));
+    final byte[] spanBytes = Codec.JSON.writeSpans(asList(spans));
```

Comment:
if you only need <10, then probably TestObjects.TRACE will simplify this even further (as it is a list of 3)